### PR TITLE
[ONNX] concat operator added

### PR DESCRIPTION
**Pull request by Herald-of-Aurora**

### CODE REVIEW

**Comment by zhichao-li** [comment]
any update?

**Comment by zhichao-li** [comment]
Add a prefix `ONNX` to the title

File: pyzoo/zoo/pipeline/api/onnx/mapper/concat.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+class ConcatMapper(OperatorMapper):
+    def __init__(self, node, _params, _all_tensors):
+        super(ConcatMapper, self).__init__(node, _params, _all_tensors)
+
+    def _to_tensor(self):
+        axis = -1
+        if "axis" in self.onnx_attr.keys():
+            axis = int(self.onnx_attr['axis'])
+
+        data = self.model_inputs[0].zvalue
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/concat.py]: `concat` should accept multiple inputs instead of only the first one?

**Comment by Herald-of-Aurora** [comment]
Jenkins passed, id: 1153. (/job/ZOO-PR-Validation/1153/)

**Pull request merged**